core/src/test/java/edu/ucr/cs/riple/core/ArgumentNullableFlowTest.java

```diff
@@ -44,27 +44,32 @@ public void nullableFlowDetectionEnabledTest() {
     coreTestHelper
```

Comment:
So, should the test cases requested [here](https://github.com/nimakarimipour/NullAwayAnnotator/pull/53#discussion_r961121779) be part of this PR or are they pending some other modification from the other half of the PR? (Note that I am interested in seeing what would happen with those simple test cases, without necessarily adding five return null branches like suggested in #62. Happy to see both types of test cases, but the simple cases would be illustrative)

Reply:
Yes that case is also covered in this approach. It has an accurate mechanism for updating states after each injection or within measurements. Added the unit test :) 838fad7784107ccf52af2f84257983976459db50